tests/screenplay.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable no-undef*/
+/* eslint-disable no-unused-expressions*/
+/* eslint-disable func-names */
+const chai = require('chai');
+const moderator = require('../src/moderator');
+
+const expect = chai.expect;
+
+describe('Screenplay', () => {
+    describe('fromGame()', () => {
```

Comment:
Something like that was my original intent but there's a catch:

`witch`, `cult_leader`, and `villager` are roleKeys, not callKeys.

In this particular example, the roles `cult_leader` and `witch` are bound each to one call whose key is the same roleKey, but in the case of some roles like `mason`, `cupid`, `fruit_brute`, and some other wolves they are not only  not bound to a single call, but their calls don't necessarily match their respective roleKeys.

To be able to test the actual values (and not only the structure) of the resulting screenplay, and have a test that's not insanely brittle, I'd have to mock or inject or whatever else to be able to properly assert upon those values provided on the test and not on something foreign to the test.